Example/IceCream_Example/DevelopersViewController.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  DevelopersViewController.swift
+//  IceCream_Example
+//
+//  Created by Soledad on 2019/4/13.
+//  Copyright © 2019 蔡越. All rights reserved.
+//
+
+import UIKit
+import RealmSwift
+import RxRealm
+import RxSwift
+
+class DevelopersViewController: UIViewController {
+
+    var developers: [Person] = []
+    let bag = DisposeBag()
+    
+    let realm = try! Realm()
+    
+    lazy var addBarItem: UIBarButtonItem = {
+        let b = UIBarButtonItem(barButtonSystemItem: UIBarButtonSystemItem.add, target: self, action: #selector(add))
+        return b
+    }()
+    
+    lazy var tableView: UITableView = {
+        let tv = UITableView()
+        tv.register(UITableViewCell.self, forCellReuseIdentifier: "cell")
+        tv.delegate = self
+        tv.dataSource = self
+        return tv
+    }()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        view.addSubview(tableView)
+        navigationItem.rightBarButtonItem = addBarItem
+        title = "Developers"
+        
+        bind()
+    }
+    
+    override func viewWillLayoutSubviews() {
+        super.viewWillLayoutSubviews()
+        tableView.frame = view.frame
+    }
+    
+    func bind() {
+        let realm = try! Realm()
+        
+        /// Results instances are live, auto-updating views into the underlying data, which means results never have to be re-fetched.
+        /// https://realm.io/docs/swift/latest/#objects-with-primary-keys
+        let developers = realm.objects(Person.self)
+        
+        Observable.array(from: developers).subscribe(onNext: { (developers) in
+            /// When developers data changes in Realm, the following code will be executed
+            /// It works like magic.
+            self.developers = developers.filter { !$0.isDeleted }
+            self.tableView.reloadData()
+        }).disposed(by: bag)
+    }
+    
+    @objc func add() {
+        let user = Person()
+        user.name = "Yue Cai"
```

Comment:
What's interesting is that the first time I ran this I got "Jim" added to my Realm. I see that it's defaulted to Jim, but I don't really see how that wasn't overridden given that this is how we add the new users.